### PR TITLE
fully updating PT-BR translation

### PR DESCRIPTION
**Pull request by Lu-Brito**
fully updating PT-BR translation

### CODE REVIEW

File: locale/pt_BR/manager.po [hunk 1]
```diff
@@ -171,13 +171,13 @@ msgid "manager.people.enrollSyncJournal"
...
 msgid "manager.people.mergeUsers.from.description"
-msgstr "Escolha um usuário para mesclar seu cadastro dentro de outra conta (quando um usuário possui duas contas distintas, em vez de acumular funções editoriais). O cadastro selecionado primeiro será excluído e suas submissões e histórico de tarefas atribuías ao segundo cadastro escolhido."
+msgstr "Escolha um usuário para mesclar seu cadastro dentro de outra conta (quando um usuário possui duas contas distintas, em vez de acumular funções editoriais). O cadastro selecionado primeiro será excluído e suas submissões e histórico de tarefas atribuídas ao segundo cadastro escolhido."
 
 msgid "manager.people.mergeUsers.into.description"
 msgstr "Escolha um usuário a quem deseja atribuir as submissões e histórico de tarefas do cadastro anterior."
 
 msgid "manager.people.roleEnrollment"
-msgstr "Papel de {$role}"
+msgstr "Papel dos usuários"
```
Comment (asmecher) [inline locale/pt_BR/manager.po]: @Lu-Brito, the English translation does make use of the `{$role}` variable -- that should be maintained for consistency with other translations.

**Comment by asmecher** [comment]
@Lu-Brito, I flagged one thing on this PR needing a change. Also, if you're interested in using Weblate for translations, please let me know -- we are in the process of launching our new translation toolset.

File: locale/pt_BR/manager.po (hunk 1 above)
Comment (Lu-Brito) [reply locale/pt_BR/manager.po]: @asmecher I already did the changes, but I couldn't do the pull request. Could you help me? How can I do it? Thank you!

Comment (asmecher) [reply locale/pt_BR/manager.po]: Usually you'd commit the change to your existing branch, then push your branch up to your Github account. That should automatically add the change to this PR.

Comment (Lu-Brito) [reply locale/pt_BR/manager.po]: So, I'm asking exactly why I've already committed the change to my repository and the changes didn't automatically appear in PR. Or did they show up? Thank you @asmecher!

Comment (asmecher) [reply locale/pt_BR/manager.po]: @Lu-Brito, it does show up -- as a second commit in the same PR: https://github.com/pkp/ojs/pull/2563/commits/92d375865ca0a44698e8ef3f015abc06ee90982b
I'll merge this, thanks!

**Pull request merged**